### ISSUE TITLE
E2E: [sig-scheduling] SchedulerPreemption [Serial] validates basic preemption works [Conformance]

### ISSUE THREAD

**Issue by RobertKielty** [kubernetes/kubernetes#96955]
**Which jobs are flaking**:
sig-release-master-blocking Conformance - GCE - master
**Which test(s) are flaking**:
E2E: [sig-scheduling] SchedulerPreemption [Serial] validates basic preemption works [Conformance]

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-blocking#Conformance%20-%20GCE%20-%20master&exclude-non-failed-tests=&width=5

**Reason for failure**:
```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:629
Nov 28 15:03:04.393: Unexpected error:
    <*errors.errorString | 0xc0002001f0>: {
        s: "timed out waiting for the condition",
    }
    timed out waiting for the condition
occurred
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/scheduling/predicates.go:932
```
**Anything else we need to know**:
- go.k8s.io/triage is currently unavailable  
- links to specific failures in spyglass appreciated
https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-gce-conformance-latest/1332676160026316800

From TestGrid alert ...

> Test [sig-scheduling] SchedulerPreemption [Serial] validates basic preemption works [Conformance] has failed 1 times . It passed and failed at commit/5ed4b76a0. Earliest failure found at commit/5ed4b76a0 (Sat Nov 28 05:22 PST).

**Comment by Huang-Wei** [comment]
I cannot spot any significant logic flaw in the test. A possible cause is the timeout is set too short (1 minute), which _may be_ not enough for the terminating pod to run. Notice that unlike integration test, the time includes the end to end cycle: (1) preemptor being scheduled and evict low-priority pods, (2) low-priority pods being terminating -> reported by kubelet that the victims have been deleted, (3) scheduler gets notifications from API server that the victims have been deleted, (4) preemptor gets scheduled, (5) kubelet reports the preemptor is running.

/cc @damemi @ahg-g @alculquicondor for more thoughts.

**Comment by pacoxu** [comment]
Maybe, some logs to `describe the pod`/`list pod in that ns` would help. 

Just making the timeout longer don't make sense.

**Comment by ahg-g** [comment]
Is https://github.com/kubernetes/kubernetes/pull/96807 addressing this flake?

@ravisantoshgudimetla @damemi

**Comment by Huang-Wei** [comment]
I think this flake happens before the failure #96807 tries to fix:
 https://github.com/kubernetes/kubernetes/blob/98bc258bf5516b6c60860e06845b899eab29825d/test/e2e/scheduling/preemption.go#L181

**Comment by alculquicondor** [comment]
That's indeed a lot of steps, and flakiness seems very low. I think increasing the timeout for preemption seems reasonable.

**Comment by Huang-Wei** [comment]
@RobertKielty is this flake a must-fix in 1.20?
